web-react-ts/src/pages/services/banking/anagkazo/ConfirmAnagkazoBanking.tsx

```diff
@@ -0,0 +1,222 @@
+import React, { useContext, useState, useEffect } from 'react'
+import { Button, Card, Container, Spinner, Table } from 'react-bootstrap'
+import { MemberContext } from 'contexts/MemberContext'
+import { ChurchContext } from 'contexts/ChurchContext'
+import PlaceholderCustom from 'components/Placeholder'
+import { getWeekNumber } from 'jd-date-utils'
+import { STREAM_BANKING_DEFAULTERS_LIST } from 'pages/services/defaulters/DefaultersQueries'
+import { useQuery, useMutation } from '@apollo/client'
+import ApolloWrapper from 'components/base-component/ApolloWrapper'
+import { Formik, Form, FormikHelpers } from 'formik'
+import FormikControl from 'components/formik-components/FormikControl'
+import { useNavigate } from 'react-router-dom'
+import usePopup from 'hooks/usePopup'
+import { CONFIRM_BANKING } from './Treasury.gql'
+import CloudinaryImage from 'components/CloudinaryImage'
+import { ServiceContext } from 'contexts/ServiceContext'
+import { DISPLAY_FELLOWSHIP_SERVICE } from 'pages/services/record-service/RecordServiceMutations'
+import { alertMsg, throwErrorMsg } from 'global-utils'
+import Popup from 'components/Popup/Popup'
+
+type FormOptions = {
+  defaulterSearch: string
+}
+
+type Defaulter = {
+  bacenta: {}
+  id: string
+  leader: { pictureUrl: string; fullName: string }
+  meetingDay: { day: string }
+  name: string
+  services: [{}]
+  __typename: string
+}
```

Comment:
Will have to review this type again later. Not sure about the bacenta dad services and __typename

Typename should have been ChurchLevel